packages/rancher-alerting-drivers/charts/templates/cluster-role.yaml

```diff
@@ -0,0 +1,66 @@
+{{- $admin := "alerting-drivers-admin" -}}
+{{- $edit  := "alerting-drivers-edit"  -}}
+{{- $view  := "alerting-drivers-view"  -}}
+
+{{- if not (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" $admin ) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ $admin }}
+  labels:
+    {{- include "rancher-alerting-drivers.labels" . | nindent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-admin: "true"
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    resourceNames:
+      - {{ include "rancher-alerting-drivers.sachetFullname" . }}
+      - {{ include "rancher-alerting-drivers.prom2teamsFullname" . }}
+    verbs:
+      - "*"
+{{- end  }}
+---
+{{- if not (lookup "rbac.authorization.k8s.io/v1" "ClusterRole" "" $edit ) }}
+apiVersion: rbac.authorization.k8s.io/v1
+kind: ClusterRole
+metadata:
+  name: {{ $edit }}
+  labels:
+    {{- include "rancher-alerting-drivers.labels" . | nindent 4 }}
+    rbac.authorization.k8s.io/aggregate-to-edit: "true"
+rules:
+  - apiGroups:
+      - ""
+    resources:
+      - configmaps
+    resourceNames:
+      - {{ include "rancher-alerting-drivers.sachetFullname" . }}
+      - {{ include "rancher-alerting-drivers.prom2teamsFullname" . }}
+    verbs:
+      - "*"
```

Comment:
This means any project member can edit these config maps, that ok ?

Reply:
I followed the pattern of how Monitoring sets the permissions: monitoring-admin and monitoring-edit has the same set of permissions

Reply:
@aiyengar2 can you confirm this. I suppose if we are going project by project then we have to.